### DISCUSSION TITLE
Network Firewall

### DISCUSSION THREAD

**Discussion by funkymokey** [3kh0/ext-remover#538] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is there any possible way to disable a firewall on a wifi? (preferably school)

**Comment by obkduishinijf** [comment, marked as answer]
no, unless you have unrestricted access to the network's settings

**Reply by sillyangel** [reply]
question, how did you unroll ur hcpl.net computer i have one too, plus i also have a usb drive. please and thank u

**Reply by obkduishinijf** [reply]
> question, how did you unroll ur hcpl.net computer i have one too, plus i also have a usb drive. please and thank u

that wasn't mine, that was someone else's, all i said was what to do with it. also, just use sh1mmer

**Reply by funkymokey** [reply]
> question, how did you unroll ur hcpl.net computer i have one too, plus i also have a usb drive. please and thank u

shimmer of course

**Reply by obkduishinijf** [reply]
> > question, how did you unroll ur hcpl.net computer i have one too, plus i also have a usb drive. please and thank u
> 
> shimmer of course

sh1mmer is always the best option for unenrolling

**Reply by sillyangel** [reply]
i have torgdor

**Comment by CheezeKake77** [comment]
If your school doesn't block extensions, I'm pretty sure you can use proxy switchers to do this. Might be thinking of something else though.

**Comment by viperbot2000** [comment]
sh1mmer.me has a possible awnser
